Makefile

```diff
@@ -211,18 +211,10 @@ tools/bin/xprog: tools/check/xprog.go
 	cd tools/check; \
 	$(GO) build -o ../bin/xprog xprog.go
 
-tools/bin/megacheck: tools/check/go.mod
-	cd tools/check; \
-	$(GO) build -o ../bin/megacheck honnef.co/go/tools/cmd/megacheck
-
 tools/bin/revive: tools/check/go.mod
 	cd tools/check; \
 	$(GO) build -o ../bin/revive github.com/mgechev/revive
 
-tools/bin/goword: tools/check/go.mod
```

Comment:
How about the others like `revive` and `unconvert`?

Reply:
```unconvert``` can be removed but revive cannot be removed now. because revive has many problems and are not able to find the code style issue. so we disable the ```revive``` in the bazel nogo at the part folder. we still use it to check the code in the other part folder.